keyboards/cbkbd/coffeevan/keymaps/default/keymap.c

```diff
@@ -0,0 +1,42 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Minisub style split-space
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬──────┐
+     * │Esc│ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ Bspc │
+     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬─────┤
+     * │ Tb │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; | Ent │
+     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬───┤
+     * │ Shft │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │Sft│
+     * ├────┬─┴─┬─┴──┬┴───┴───┴─┬─┴───┴──┬┴───┼───┼───┼───┤
+     * │Ctrl|Spr│ Al │ Spacebar |Spacebar│Meta│Alt│Spr│Ctl│
+     * └────┴───┴────┴──────────┴────────┴────┴───┴───┴───┴
+     *
+     * JetVan style 6.25u
+     * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬──────┐
+     * │Esc│ Q │ W │ E │ R │ T │ Y │ U │ I │ O │ P │ Bspc │
+     * ├───┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬─────┤
+     * │ Tb │ A │ S │ D │ F │ G │ H │ J │ K │ L │ ; | Ent │
+     * ├────┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬───┤
+     * │ Shft │ Z │ X │ C │ V │ B │ N │ M │ , │ . │ / │Sft│
+     * ├────┬─┴─┬─┴──┬┴───┴───┴───┴───┴───┴───┼───┼───┼───┤
+     * │Ctrl|Spr│ Al │        Spacebar        │Alt│Spr│Ctl│
+     * └────┴───┴────┴────────────────────────┴───┴───┴───┴
+     */
```

Comment:
If the intent here is to showcase the available layouts, that would be better suited in a `matrix_diagram.md` file. There are various [examples](https://github.com/qmk/qmk_firmware/blob/master/keyboards/cannonkeys/gentoo/matrix_diagram.md?plain=1) within the repo.
```suggestion
```

Reply:
Thanks, I'll give that a look